notifications_api_common/models.py

```diff
@@ -0,0 +1,120 @@
+from typing import Optional
+
+from django.contrib.postgres.fields import ArrayField
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from solo.models import SingletonModel
+from zds_client import Client, ClientAuth
+from zgw_consumers.constants import APITypes
+from zgw_consumers.service import Service
+
+
+class NotificationsConfig(SingletonModel):
```

Comment:
I'd also add a custom manager that automatically performs a `select_related("notifications_api_service")` to the `get_queryset`, that saves a DB query whenever this config used.